assets/js/components/ExecutionResults/ChecksResultFilters.jsx

```diff
@@ -4,14 +4,25 @@ import Filter from '@components/Table/Filter';
 
 export const RESULT_FILTER_FIELD = 'result';
 
-function ChecksResultFilters({ onChange }) {
+const getFilters = (savedFilters, searchParams) =>
+  savedFilters &&
```

Comment:
If I am getting it right in the worst case `savedFilters` comes as an empty array, so this initial check might be not necessary.

Reply:
Could come as undefined, updated it to have a default prop :+1: